.gitlab/ci/on-push.yml

```
@@ -88,11 +88,64 @@ create-instances:
       [ -n "${TIME_TO_LIVE}" ] && TTL=${TIME_TO_LIVE} || TTL=300
       python3 ./Tests/scripts/awsinstancetool/aws_instance_tool.py -envType "$IFRA_ENV_TYPE" -timetolive $TTL -outfile "$ARTIFACTS_FOLDER/env_results.json"
     - section_end "Create Instances"
-    - section_start "Upload Artifacts to GCP"
+    - section_start "Upload Artifacts to GCP" --collapsed
     - ./Tests/scripts/upload_artifacts.sh
     - section_end "Upload Artifacts to GCP"
 
 
+create-instances-mpv2:
+  tags:
+    - gke
+  extends:
+    - .default-job-settings
+  rules:
+    - if: '$CI_PIPELINE_SOURCE =~ /^(push|contrib)$/'
+    - if: '$NIGHTLY'
+    - if: '$INSTANCE_TESTS'
+  cache:
+    policy: pull-push
+  needs: []
+  stage: create-instances
+  script:
+    - !reference [.download-demisto-conf]
+    - !reference [.create-id-set-mp-v2]
+    - section_start "Create Release Notes and Common Server Documentation" --collapsed
+    - echo "Creating Release Notes and Content Descriptor"
+    - python3 Utils/release_notes_generator.py $CONTENT_VERSION $GIT_SHA1 $CI_BUILD_ID --output $ARTIFACTS_FOLDER/packs-release-notes.md --github-token $GITHUB_TOKEN
+    - cp content-descriptor.json $ARTIFACTS_FOLDER
+    - ./Documentation/commonServerDocs.sh
+    - section_end "Create Release Notes and Common Server Documentation"
+
+    - section_start "Create Content Artifacts and Update Conf" --collapsed
+    - demisto-sdk create-content-artifacts -a $ARTIFACTS_FOLDER --cpus 8 --content_version $CONTENT_VERSION --marketplace "marketplacev2" >> $ARTIFACTS_FOLDER/logs/create_content_artifacts.log
+    - gcloud auth activate-service-account --key-file="$GCS_ARTIFACTS_KEY"  # Not sure if needed:
+    - zip -j $ARTIFACTS_FOLDER/uploadable_packs_mpv2.zip $ARTIFACTS_FOLDER/uploadable_packs/* >> $ARTIFACTS_FOLDER/logs/zipping_uploadable_packs.log || ((($? > 0)) && echo "failed to zip the uploadable packs, ignoring the failure")
+    - rm -rf $ARTIFACTS_FOLDER/uploadable_packs
+    - cp "./Tests/conf.json" "$ARTIFACTS_FOLDER/conf.json"
+    - section_end "Create Content Artifacts and Update Conf"
+
+    - section_start "Calculate Packs Dependencies" --collapsed
+    - python3 ./Tests/Marketplace/packs_dependencies.py -i ./Tests/id_set.json -o $ARTIFACTS_FOLDER/packs_dependencies.json
```

```suggestion
    - demisto-sdk find-dependencies -idp Tests/id_set.json --all-packs-dependencies -o $ARTIFACTS_FOLDER/packs_dependencies.json
```
We should use the version of the dependencies calculation that was migrated to sdk. 
On top of that, need to remember for future efficiency and performance improvements to try and save the dependencies graph calculated in the id set creation (line 111) to be used here as well.